v2.6/getting-started/kubernetes/installation/hosted/canal/index.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Canal Hosted Install
+canonical_url: 'https://docs.projectcalico.org/v3.0/getting-started/kubernetes/installation/hosted/kubeadm/'
+---
+
+## About Canal
+
+Canal allow users to easily deploy Calico and flannel networking
+together as a unified networking solution - combining Calico’s
```

Comment:
In this case, it should be an em dash with no surrounding spaces...not a hyphen. For our particular situation, our renderer, you should just paste in the em dash character. Here it is for your convenience `—`